### ISSUE TITLE
Start error

### ISSUE THREAD

**Issue by andris155** [ImagicalMine/ImagicalMine#231433]
- [x] This issue isn't related to any plugins or plugin related crashes.
- [x] This issue isn't duplicated - you can check if it is by using the search bar located at the top left hand corner and select "Issues" on the left.
- [x] This issue includes appropriate markdown for sections - e.g. code blocks for crash dumps.
- [x] This issue is understandable - feel free to use your native language to write issues if you are not comfortable with English.
- [x] This issue matches the contributing guidelines - the link is just above the title field.

## Reproducing the issue
Start latest dev. im.

## Client information
latest dev im., php 7.0.1, Windows 10

## Optional information
```
ImagicalMine Crash Dump Sat Apr 23 12:25:00 BST 2016

Error: syntax error, unexpected 'case' (T_CASE)
File: /src/pocketmine/Player
Line: 3048
Type: notice

Code:
[3039] 					}
[3040] 				elseif($recipe instanceof ShapelessRecipe){
[3041] 					$recipe2 = new ShapelessRecipe($packet->output[0]);
[3042] 					foreach($recipe2->getIngredientList() as $content){
[3043]  							if($this->getInventory()->contains($content)) $packet->input[] = $content;
[3044] 					}
[3045] 				}
[3046]  			}
[3047]  			break;
[3048] 		case ProtocolInfo::CRAFTING_EVENT_PACKET:
[3049] 			if ($this->spawned === false or !$this->isAlive()) {
[3050] 				break;
[3051] 			}elseif (!isset($this->windowIndex[$packet->windowId])) {
[3052] 				$this->inventory->sendContents($this);
[3053] 				$pk = new ContainerClosePacket();
[3054] 				$pk->windowid = $packet->windowId;
[3055] 				$this->dataPacket($pk);
[3056] 				break;
[3057] 			}
[3058] 			$recipe = $this->server->getCraftingManager()->getRecipe($packet->id);

Backtrace:
#0 /src/pocketmine/Server(570): spl_autoload_call(string pocketmine\Player)
#1 /src/pocketmine/Server(2365): pocketmine\Server::getGamemodeString(integer 0)
#2 /src/pocketmine/Server(1949): pocketmine\Server->start(boolean)
#3 /src/pocketmine/PocketMine(512): pocketmine\Server->__construct(pocketmine\CompatibleClassLoader object, pocketmine\utils\MainLogger object, string D:\Minecraft Szerverek\ImagicalMine\, string D:\Minecraft Szerverek\ImagicalMine\, string D:\Minecraft Szerverek\ImagicalMine\plugins\)

ImagicalMine version: 1.4 #0 [Protocol 46; API 2.0.0]
Git commit: 0000000000000000000000000000000000000000
uname -a: Windows NT DESKTOP- 10.0 build 10586 (Windows 10) i586
PHP Version: 7.0.1
Zend version: 3.0.0
OS : WINNT, win

```

**Comment by dterweij** [comment]
Fixed by commit 2bb1ab2afaf1dfef52c2aa60ec1288d1f8724742

Issue can be closed.

**Comment by andris155** [comment]
Thanks!
